t/40overwrite_modifications.t

```diff
@@ -43,8 +34,37 @@ lives_ok {
     dump_schema( overwrite_modifications => 1 );
 } 'does not throw when dumping with overwrite_modifications';
 
+# Replace the md5 with a bad MD5 in Foo.pm
+my $foopm_content = slurp_file($foopm);
+my ($md5) = $foopm_content =~/md5sum:(.+)$/m;
+# This cannot be just any arbitrary value, it has to actually look like an MD5
+# value or DBICSL doesn't even see it as an MD5 at all (which makes sense).
+my $bad_md5 = reverse $md5;
+rewrite_foopm($foopm, qr{md5sum:.+$}, "md5sum:$bad_md5");
 
-unlike slurp_file $foopm, qr/"somethingelse"/, "Modifications actually overwritten";
```

Comment:
Having had a closer look this morning, it's clear that that test should still be there, so I've reinstated it.